docs/infrastructure/self-hosted/self-hosted-requirements.md

```diff
@@ -217,9 +219,52 @@ The following is an example architecture for the installation of the Mia-Platfor
 | Google Cloud Storage      |   SaaS      |                |      |      |          |
 | Google Cloud Platform KMS |   SaaS      |                |      |      |          |
 
-### Designating the Console Super User
 
-When installing an instance of Mia-Platform Console, a user with the Console Super User role will be added to the database automatically.
+## Runtime requirements 
+
+Applications developed with Mia-Platform Console will run in a Kubernetes environment. This Kubernetes runtime environment will be managed and monitored by Mia-Platform Console.  
+In order to do this, the following requirements must be fulfilled for a self-hosted installation.
+
+### Self-Hosted runtime installation prerequisites
+
+The following tools will be installed in your runtime Kubernetes clusters if desired:  
+
+* (Optional) Mia Monitoring stack
+* (Optional) Mia Logging stack
+* Traefik Ingress Controller
+* (Optional) Velero for Disaster Recovery
+
+:::note
+These are minimum requirements for your cluster. These requirements have a scale factor that must be taken in account when estimating your cluster size.
+:::
+
+| **Tool**                     | **Namespace CPU limits** | **Namespace RAM limits** | **Scales with**                                               |
+|------------------------------|--------------------------|--------------------------|---------------------------------------------------------------|
+| Monitoring stack             | 1 core                   | 3 GiB                    | Number of cluster nodes, number of metrics                    |
+| Logging stack                | 0.5 core                 | 1 GiB                    | Number of cluster nodes, number of applications logs, traffic |
+| Traefik Ingress Controller   | 0.3 core                 | 300 MiB                  | Traffic                                                       |
+| Velero for Disaster Recovery | 0.2 core                 | 400 MiB                  | -                                                             |
+
+In total, the minimum requirements in order to host all the above tools in the Kubernetes cluster are about 2 cores and 5 GiB of memory. Above values will scale with the number of nodes, traffic, applications.
+
+Velero, in addition, needs a S3-like bucket storage for keeping the Kubernetes cluster backup.
+
+Grafana Mimir will be installed in the same Kubernetes cluster hosting Mia-Platform Console. The minimum requirements needed for Grafana Mimir are 2 cores and 4 GiB of memory.
```

Comment:
Why don't we insert Grafana Mimir as optional component in the table above, explaining why you should install Mimir for?

Reply:
I don't get the problem. Do you think that line 252 is not clear in that way?

Reply:
I mean that Mimir is missing in the table above. The user that reads that table think "Ok, with 2 cores I'm ok with the monitoring stack" but it's not clear, IMHO, that with Mimir you will need 2 more cores and why you should install Mimir too.

I think we can Add "Grafana Mimir" in the table above (that starts at line 242) as Optional component, to simplify the user calculation of the needed CPU.
Moreover, outside of the table, we can provide a little explanation like "Mimir is suggested to as a long-term storage for the prometheus metrics in a more solid way .......".